### ISSUE TITLE
Screen Reader: Many Pages: The control opens another page/URL but announces with a button role

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#76938]
## Action Performed:
Prerequisite:
- the user is logged in.

1. Using JAWS + Chrome, open the page https://new.expensify.com/settings/profile
2. Navigate using TAB to the Expensify logo, at the top, left corner of the page
3. Observe how it is announced

Other Occurrences:
1. On Workspaces: the row of the workspace

## Expected Result:
The Expensify logo logo will be announced as a link, as it takes the user to a new page

## Actual Result:
When users navigate to the Expensify logo at the top, left corner of the screen, it is announced as a 'Button'. Users will become confused if they are unexpectedly taken to a different page

HTML:
<button aria-label="Home" role="button" data-selection-scrapper-hidden-element="true" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-9ffhgg r-lrvibr" data-testid="ExpensifyLogoButton" type="button" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" id="expensify-app-icon_svg__Layer_1" x="0" y="0" viewBox="0 0 48 48" width="100%" height="100%" hovered="false" pressed="false" style="height: 40px; width: 40px; border-radius: 40px; background-color: rgb(230, 225, 218);"><style>.expensify-app-icon_svg__st1{fill:#002e22}</style><path d="M0 0h48v48H0z" style="fill: rgb(3, 212, 124);"></path><path d="M30.6 18.5v-3.9H17.4v18.9h13.2v-3.9h-8.4V26H29v-4h-6.7v-3.4h8.3z" class="expensify-app-icon_svg__st1"></path><path d="M24 8.3c3.3 0 6.4 1 9 2.8.6.4 1.5.4 2.1-.1.6-.6.6-1.7-.1-2.2-3.1-2.2-6.9-3.5-11-3.5C13.7 5.3 5.3 13.7 5.3 24c0 4.1 1.3 7.9 3.5 10.9.5.7 1.6.8 2.2.1.6-.6.6-1.4.1-2.1-1.8-2.6-2.8-5.6-2.8-9 0-8.5 7.1-15.6 15.7-15.6m15.5 5.1c-.5-.7-1.6-.8-2.2-.2s-.6 1.4-.2 2.1c1.6 2.5 2.6 5.5 2.6 8.7s-1 6.3-2.7 8.8c-.4.7-.4 1.5.2 2.1s1.7.6 2.2-.1c2.1-3 3.4-6.8 3.4-10.8s-1.2-7.6-3.3-10.6M24 39.7c-3.2 0-6.2-1-8.7-2.6-.7-.4-1.5-.4-2.1.2s-.6 1.7.2 2.2c3 2.1 6.7 3.3 10.6 3.3s7.7-1.3 10.8-3.4c.7-.5.8-1.6.1-2.2-.6-.6-1.4-.6-2.1-.2-2.5 1.7-5.5 2.7-8.8 2.7" class="expensify-app-icon_svg__st1"></path></svg></button>

## Area issue was found in:
Settings–Profile

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Implement the control as a native <a> element with a valid href attribute if it opens another page or URL,
https://www.magentaa11y.com/#/web-criteria/component/link?tab=2
https://a11y-101.com/design/button-vs-link

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1914" height="998" alt="Image" src="https://github.com/user-attachments/assets/2d85c3f0-828c-45e5-a73d-a0a66926be49" />

https://github.com/user-attachments/assets/8a0f3a5c-a1ff-4765-a13a-ca16ebff6005

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6994132
**Issue reported by:** NA
**Slack conversation:** NA

<details><summary>Issue Owner</summary>Current Issue Owner: @jjcoffee</details>

**Comment by MelvinBot** [comment]
🤖 Draft PR created: https://github.com/Expensify/App/pull/82754
Please mark it as "Ready for review" when it is ready for review.

**Comment by jjcoffee** [comment]
Hmm I'm not sure if this issue is really an issue. It _does_ present visually as a button, and it navigates _within_ the app without a page refresh, so why should it be announced as a link? 

I'd imagine links here refers to actual external resources that we'd also expect to visually look like links (the footer links on the sign-in page, for example), rather than anything that navigates (otherwise probably all our buttons would have to have their role changed to links, since there's usually some navigation involved!).
